dotnet/src/SemanticKernel/SkillDefinition/SkillCollection.cs

```diff
@@ -15,8 +15,8 @@ namespace Microsoft.SemanticKernel.SkillDefinition;
 /// The class holds a list of all the functions, native and semantic, known to the kernel instance.
 /// The list is used by the planner and when executing pipelines of function compositions.
 /// </summary>
-[System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1710:Identifiers should have correct suffix", Justification = "It is a collection")]
-[System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1711:Identifiers should not have incorrect suffix", Justification = "It is a collection")]
+[SuppressMessage("Naming", "CA1710:Identifiers should have correct suffix", Justification = "It is a collection")]
```

Comment:
nit: is CA170 required? I saw in other review that it wasn't needed. Unrelated, feel free to resolve.

Reply:
I've been getting both warnings, very confusing, so I added both (this is old code)